lib/WebService/Stripe.pm

```diff
@@ -173,6 +173,12 @@ method cancel_transfer(Str $id, :$headers) {
     return $self->post("/v1/transfers/$id/cancel", undef, headers => $headers);
 }
 
+method create_reversal($xfer_id, HashRef $data = {}, HashRef $opts = {}) {
```

Comment:
Ah yeah... that's kinda clunky. I like this better. I'm just not liking that it's inconsistent with all of the other methods. Should I get over that? :)